pkgs/top-level/lua-packages.nix

```diff
@@ -215,6 +215,30 @@ let
     '';
   };
 
+  luaiconv = buildLuaPackage rec {
+    name = "luaiconv-${version}";
+    version = "7";
+    src = fetchFromGitHub {
+      owner = "ittner";
+      repo = "lua-iconv";
+      rev = "e8d34024a6b185a759733915f116cc5588550261";
+      sha256 = "0rd76966qlxfp8ypkyrbif76nxnm1acclqwfs45wz3972jsk654i";
+    };
+
+    buildInputs = [];
```

Comment:
Delete this line.

Reply:
If you need this and the latter `preBuild` to override `buildLuaPackage` defaults, just don't use `buildLuaPackage`.

Reply:
ack

Reply:
Sorry I don't understand, I am really just following the convention of the rest of this file. I will ofcourse delete `buildInputs` but I don't understand why `buildLuaPackage` should be dropped :confused:

Reply:
Look at the definition of `buildLuaPackage`: https://github.com/NixOS/nixpkgs/blob/b5d11a76/pkgs/development/lua-modules/generic/default.nix
Basically it's `mkDerivation` with custom `preBuild` and `buildInputs`. I was not familiar with lua packaging and it appeared that you were attempting to circumvent its utility by overriding both of these attributes. Sorry.

Reply:
Ahh I see okay, well I've left `buildLuaPackage` etc in tact, but have dropped the `buildInputs` line, moved the `meta` section to the bottom and added `homepage` :cat: